vlm/src/main/resources/reference.conf

```diff
@@ -1,3 +1,3 @@
 vlm.geotiff.s3 {
-  allowGlobalRead: false
-}
\ No newline at end of file
+  allow-global-read: false
```

Comment:
Rather than doing this, let's keep it as `allowGlobalRead: false` (as we have this style in `geotrellis-gdal`):

```scala
case class S3Options(allowGlobalRead: Boolean)

object S3Options extends PureConfigSettings with Serializable {
  val config = pureconfig.loadConfigOrThrow[S3Options]("vlm.geotiff.s3")
}
```

^ it's the current `Config.scala`. I think we need to rename it and move into `config` package.

Reply:
Doesn't pureconfig require snake case for the config variables?

Reply:
@echeipesh it's configurable O:

Reply:
Well, we can just come up with some common style and use it everywhere :)

Reply:
@echeipesh also renaming this file would be very helpful as well :D dunno when it was merged.

Reply:
We're making hyphen case as default.

Reply:
Since it's used by the most libraries (for instance akka-http)